Mathlib/MeasureTheory/Group/Action.lean

```diff
@@ -167,6 +169,19 @@ theorem map_smul : map (c • ·) μ = μ :=
 
 end MeasurableSMul
 
+theorem MeasurePreserving.smulInvariantMeasure_iterateMulAct
```

Comment:
Any reason not to add:
```suggestion
@[to_additive]
theorem MeasurePreserving.smulInvariantMeasure_iterateMulAct
```

Reply:
The only reason is me forgetting to do it.